SeaBattle/Interfaces/IInput.cs

```diff
@@ -5,10 +5,10 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace SeaBattle.Input.inputHandler
+namespace SeaBattle.Engine.Interfaces
 {
     public interface IInput
     {
-        public Coordinates GetCoordinates();
+        public Task<Coordinates> GetCoordinatesAsync();
```

Comment:
Issue: In your case it's kind of unnecessary, you don't have a real async input here